File: sentimental-mario-less/mario.py
# Prompt user for height of pyramid
while True:
    try:
        height = int(input("Height: "))
        if (height > 0) and (height <= 8):
            break
    except ValueError:
        print("That's not an integer!")

# print pyramid
for i in range(1, height + 1):
    print(" " * (height - i) + "#" * i)

File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt user for height
    // Reprompt if input is invalid
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while ((height <= 0) || (height > 8));

    // Print hashes and spaces
    for (int i = height - 1; i >= 0; i--)
    {
        // Print spaces
        printf("%.*s", i, "       ");
        // Print left half of pyramid
        printf("%.*s", (height - i), "#########");
        //Print spaces
        printf("  ");
        //Print right half of pyramid
        printf("%.*s", (height - i), "#########");
        //Print new row
        printf("\n");
    }

}